src/wrapper.rs

```diff
@@ -4,6 +4,8 @@ use core::fmt::{self, Debug, Display};
 #[repr(transparent)]
 pub struct MessageError<M>(pub M);
 
+// TODO add backtrace member to message error
```

Comment:
If we dont capture a backtrace in the root error then an iteration over the inner errors will not yield _any_ backtraces on any of the errors in the chain, 

with the changes above when serializing the inner errors we get:

```
{
  "msg": "outermost error",
  "backtrace": "disabled backtrace",
  "source": {
    "msg": "second error",
    "backtrace": "disabled backtrace",
    "source": {
      "msg": "root cause",
      "backtrace": null,
      "source": null
    }
  }
}
```

Whereas with the changes and without the backtrace in Message error we get:

```
{
  "msg": "outermost error",
  "backtrace": null,
  "source": {
    "msg": "second error",
    "backtrace": null,
    "source": {
      "msg": "root cause",
      "backtrace": null,
      "source": null
    }
  }
}
```